gitfs/utils/args.py

```diff
@@ -82,7 +82,10 @@ def check_args(self, args):
 
         # setup logging
         if args.log != "syslog":
-            handler = TimedRotatingFileHandler(args.log, when="midnight")
+            if args.log == '-':
```

Comment:
I went for `-` because it seems to be a common unix pattern. How about I allow use of both `-` and `/dev/stdout`? 
http://stackoverflow.com/a/3797806/1208936